ec2-refresh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 # Look for all ec2 instances run by user $LOGNAME and cache them in $FILE:
+region=${region-$aws}
```

Comment:
Where does the $aws variable supposed to come from?
Does `--region ""` (empty string) work as the default region?

Reply:
> Where does the $aws variable supposed to come from?

As the commit message sais:
> aws takes precedes over the region variable since we assume that if this
variable exists we are part of a larger context of run that operates on
a specific aws region.

So the answer is from a larger context such as due to calling it when running on dynamodb :slightly_smiling_face:  



> Does `--region ""` (empty string) work as the default region?

No, I simply assumed that we don't need to be rigorouse about cases where the user don't set up his environment properly. I will fix.